sql/grants.sql

```diff
@@ -0,0 +1,181 @@
+DROP PROCEDURE IF EXISTS SCHEMA_GRANTS_REFRESH;
+DELIMITER ';;'
+CREATE PROCEDURE SCHEMA_GRANTS_REFRESH(
+      IN schemaname VARCHAR(64)
+    , IN targetuser VARCHAR(190)
+    , IN privmode   VARCHAR(20)
+) BEGIN
+    -- Declare some quick variables
+    DECLARE cursor_done INT;
+    DECLARE tableName VARCHAR(64);
+    DECLARE vsql VARCHAR(4000);
+
+    -- --------------------------------------------------------------------------------------------------
+    -- PERMISSIONS
+    -- Add tables to these queries to assign grants as needed. Limit the rows by adding where clauses
+    -- against privmode
+    -- --------------------------------------------------------------------------------------------------
+    DECLARE cur_select CURSOR FOR
+        SELECT 'antispoofcache' AS tbl UNION
```

Comment:
Is the `AS tbl` necessary for these cursors?  It seems odd to include it in the cursors for SELECT, UPDATE, and INSERT but not the DELETE cursor.